src/modules/commander/commander_params.c

```diff
@@ -919,6 +919,14 @@ PARAM_DEFINE_INT32(COM_RCL_EXCEPT, 0);
  */
 PARAM_DEFINE_INT32(COM_OBS_AVOID, 0);
 
+/**
+ * Expect and require a healthy MAVLink parachute system
+ *
+ * @boolean
+ * @group Commander
+ */
+PARAM_DEFINE_INT32(COM_PARACHUTE, 0);
```

Comment:
This is kind of vague, shouldn't it be specific to arming?

`COM_ARM_PARA`? (or a better abbreviation)

Reply:
I intentionally avoided the term arm because it's meant to enable the entire MAVLink parachute communication/handling including status checks, user messaging and fail-safes in the air. Even if this pr just enables a preflight check I'm trying to avoid that we have to change the parameter name with further functionality.

Reply:
As long as it's actually going to enable/disable the parachute in the near future then that makes sense to me.

Reply:
Yes, it is. There are follow-ups necessary e.g. also for handling acknowledgments by the parachute system. Command and resending when no acknowledgement are just necessary when there is a system configured just as example.